comments/templates/django_comments_xtd/comment_tree.html

```diff
@@ -36,22 +36,29 @@
                 {% endif %}
             </div>
                {% if not item.comment.is_removed %}
-              {% if allow_flagging and item.flagged %}
+              {% if allow_flagging and item.flagged and item.comment.user != user %}
                   <a href="{% url 'comments-flag' item.comment.pk %}" class="report-comment">
                    {% translate "Report" %}
                   </a>
                   <i title="{% translate 'comment flagged' %}" class='report-comment__disclaimer'>{% translate "This is a reported comment" %} </i>
-              {% elif allow_flagging %}
+              {% elif allow_flagging and item.comment.user != user %}
               <a href="{% url 'comments-flag' item.comment.pk %}" class="report-comment">
                 {% translate "Report" %}
               </a>
               {% endif %}
-              {% if perms.comments.can_moderate %}
-                  {% if item.flagged_count %}
-                      {# Translators: Count refers to the number of people that reported a comment. #}
-                      <span class="report-comment">{% blocktranslate with item.flagged_count as count %}Reported by {{ count }}{% endblocktranslate %}</span>
-                  {% endif %}
-              {% endif %}
+                   {% if perms.comments.can_moderate %}
+                       {% if item.flagged_count %}
+                           {# Translators: Count refers to the number of people that reported a comment. #}
+                           <span class="report-comment">{% blocktranslate with item.flagged_count as count %}Reported by
```

Comment:
Better to use something like below to more generally account for pluralization in other langauges?

```
{% blocktranslate count count=item.flagged_count %}
Reported by {{ count }} user
{% plural %}
Reported by {{ count }} users
{% endblocktranslate %}
```